regexes/device/mobiles.yml

```diff
@@ -5362,13 +5367,13 @@ Lenovo:
     - regex: 'ThinkPadTablet_(A[0-9]{3})_'
       model: 'ThinkPad Tablet $1'
       device: 'tablet'
-    - regex: 'ThinkPad ?Tablet([^;/]*) Build'
+    - regex: 'ThinkPad ?Tablet([^;/]*)(?: Build)?'
```

Comment:
In General: Making the `Build` optional in such detection basically has the same effect as removing it as it will be included in the matched model afterwards... You can also see that in the failing tests

Reply:
But why do you need this build? We have users agents where "Build" is not there so this regex fail.